project-structure/standard-interfaces.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Project interface
+description: The standard interface for our projects - package.json and entrypoint
+---
+
+Our projects should all include [a `package.json` file](https://yarnpkg.com/lang/en/docs/package-json/) at the top-level of the project, which defines the standard ways the project should be run.
+
+The `package.json` file should define some [`scripts`](https://yarnpkg.com/lang/en/docs/package-json/#toc-scripts), and (usually) some [`dependencies` and `devDependencies`](https://yarnpkg.com/lang/en/docs/package-json/#toc-dependencies).
+
+## scripts
+
+These defined `scripts` can be run either with `yarn run {script-name}` or through [our local development tooling](https://canonical-web-and-design.github.io/practices/project-structure/the-run-script.html).
+
+Our standard script names:
+
+- `start` (required): The standard entrypoint for local development. It should run any required watchers for rebuilding files as they change, and run any local development servers etc..
```

Comment:
Do our projects have `start`?  If we treat it as required should we add one to project that don't have it?

Seems like currently the contents of the `scripts` are mostly dictated by run script, so they include `build`, `serve` `watch`, `test` with addition of some more specific ones. 

With this practice, do we want to enforce adding `start` as well? Is `start` supported by run script (like `./run start`)?

Reply:
Yeah this is the new standard, ad yes we should implement it on projects, and will do when we migrate to the dotrun snap. Maybe you're right that I should have held off on merging until we'd got further in rolling out the new standard.

But that this point I'd rather leave this standard in place and hurry up with my dotrun snap work.

Reply:
Sure, I don't mind. Was just surprised that the only required script is the one I haven't seen in any our project yet.